test/helper.rb

```diff
@@ -250,8 +250,10 @@ def md.directory?
       'pkginfo -l SUNWzfsr' => cmd.call('pkginfo-l-SUNWzfsr'),
       # solaris 11 package manager
       'pkg info system/file-system/zfs' => cmd.call('pkg-info-system-file-system-zfs'),
-      # dpkg-query package list
+      # dpkg-query all packages
       "dpkg-query -W -f='${db:Status-Abbrev} ${Package} ${Version}\\n'" => cmd.call('dpkg-query-W'),
+      # rpm query all packages
+      "rpm -qa --queryformat '%{NAME}   %{VERSION}-%{RELEASE}\\n'" => cmd.call('rpm-qa-queryformat'),
```

Comment:
Exactly, I played the paranoid here to avoid splitting on a space in package name or version. I'm fine with two spaces as well